tests/manage/mcg/test_write_to_bucket.py

```diff
@@ -69,9 +69,69 @@ def test_write_file_to_bucket(self, mcg_obj, awscli_pod, bucket_factory):
     @pytest.mark.polarion_id("OCS-1949")
     @tier1
     @acceptance
-    def test_data_reduction(self, mcg_obj, awscli_pod, bucket_factory):
+    def test_data_deduplication(self, mcg_obj, awscli_pod, bucket_factory):
         """
         Test data reduction mechanics
+        Args:
+            mcg_obj (obj): An object representing the current state of the MCG in the cluster
+            awscli_pod (pod): A pod running the AWSCLI tools
+            bucket_factory: Calling this fixture creates a new bucket(s)
+
+        """
+        download_dir = '/aws/deduplication/'
+        for i in range(2, 4):
+            awscli_pod.exec_cmd_on_pod(
+                command=helpers.craft_s3_command(
+                    f'cp s3://{constants.TEST_FILES_BUCKET}/danny.webm {download_dir}danny{i}.webm'
+                ),
+                out_yaml_format=False
+            )
+        bucket = bucket_factory(1)[0]
+        bucketname = bucket.name
+        full_object_path = f"s3://{bucketname}"
+        helpers.sync_object_directory(
+            awscli_pod, download_dir, full_object_path, mcg_obj
+        )
+        # TODO: new function to test any reduction at all
+        assert mcg_obj.check_data_reduction(bucketname, 43 * 1024 * 1024), (
+            'Data deduplication did not work as anticipated.'
+        )
+
+    @pytest.mark.polarion_id("OCS-1949")
+    @tier1
+    @acceptance
```

Comment:
I think this should not be part of the acceptance. @nimrod-becker @RazTamir what do you think?

Reply:
added the acceptance tag since the tag was applied to the original data reduction test

Reply:
I also suggest removing this from the acceptance suite; Since the test was originally written, it was discovered that there are inconsistencies in data reduction, and it mostly caused false failures (reduction worked, just not in the expected amount)